src/apps/workspaces/service/check_access.py

```diff
@@ -93,6 +93,9 @@ async def check_applet_respondent_list_access(self, applet_id: uuid.UUID):
         roles = [Role.OWNER, Role.MANAGER, Role.COORDINATOR, Role.REVIEWER]
         await self._check_applet_roles(applet_id, roles)
 
+    async def check_applet_manager_role(self, applet_id: uuid.UUID):
```

Comment:
I gather that the point of these functions is to _abstract away_ the user role, and so they are instead named by the _function/feature_ they're limiting access to. Mentioning "role" in the permission function name seems to violate the intended semantics. So I'd name this function something like below, what do you think?

```suggestion
    async def check_applet_assign_activity_access(self, applet_id: uuid.UUID):
```

Reply:
done